tests/Doctrine/Tests/DBAL/Driver/StatementIteratorTest.php

```diff
@@ -23,8 +24,8 @@ class StatementIteratorTest extends DbalTestCase
      */
     public function testGettingIteratorDoesNotCallFetch(string $class) : void
     {
-        /** @var IteratorAggregate|MockObject $stmt */
         $stmt = $this->createPartialMock($class, ['fetch', 'fetchAll', 'fetchColumn']);
+        assert($stmt instanceof IteratorAggregate || $stmt instanceof MockObject);
```

Comment:
I am really concerned about adding such assertions in tests:

1. If the pre-condition is not met, the test will most likely fail, there's no point in adding native PHP assertions.
2. If we want to keep the assertion, it must be done via the PHPUnit API.

In this specific test, the must be of both types (two assertions), not any of them.

Reply:
:thinking: wait I thought I removed those

Reply:
I removed it